uiplib/scrape/scrape.py

```diff
@@ -64,7 +71,7 @@ def get_reddit_image_links(url, no_of_images):
     as link of the image scraped from reddit.
     """
     # reddit requires .json format for the URL
-    page = make_json(url+'/.json')
+    page = make_json(url + '/.json')
```

Comment:
Changes in a commit need to be related. This change is not related to the commit which you made to solve an issue hence this change should be in an own commit